From 625e636e84c1eb18dd6a10aa1dade8db9f04ae48 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:36:02 -0400
Subject: [ethernet] i40e: Add missing braces to i40e_dcb_need_reconfig()

Message-id: <1409658042-22689-2-git-send-email-sassmann@redhat.com>
Patchwork-id: 90197
O-Subject: [RHEL7.1 PATCH 001/281] i40e: Add missing braces to i40e_dcb_need_reconfig()
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Dave Jones <davej@redhat.com>

Indentation mismatch spotted with Coverity.
Introduced in 4e3b35b044ea ("i40e: add DCB and DCBNL support")

Signed-off-by: Dave Jones <davej@fedoraproject.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 3d9667a9e1e463c107cb47a810ef7e85d9a31e62)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index a4b9408..b901371 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -4440,9 +4440,10 @@ bool i40e_dcb_need_reconfig(struct i40e_pf *pf,
  /* Check if APP Table has changed */
  if (memcmp(&new_cfg->app,
      &old_cfg->app,
-     sizeof(new_cfg->app)))
+     sizeof(new_cfg->app))) {
   need_reconfig = true;
   dev_info(&pf->pdev->dev, "APP Table change detected.\n");
+ }
 
  return need_reconfig;
 }
-- 
1.7.1